source/connect/authentication-connection.txt

```diff
@@ -13,3 +13,129 @@ Authentication Connection Tab
    :class: singlecol
 
 
+The :guilabel:`Authentication` tab allows you to connect deployments that 
+require authentication. To learn about authentication mechanisms within MongoDB,
+see :driver:`Authentication Mechanisms </go/current/fundamentals/auth/>`
```

Comment:
I like that you've included a link here, however I don't think we want to be linking to the Go driver docs in this case. Instead, there is a more general place we can link that provides a more general overview of the auth mechanisms. Let's link users here instead:

https://docs.mongodb.com/manual/core/authentication/#authentication-mechanisms

Also minor, but this sentence needs a period at the end of it.

Reply:
Done.